### PR TITLE
Running hybrid resync test / latest shared

### PR DESCRIPTION
**Pull request by wmathurin**
All SmartSyncJSTest's passing

### CODE REVIEW

**Comment by khawkins** [comment]
:+1:

File: tools/sdk.sh [hunk 1]
```diff
@@ -66,12 +66,6 @@ usage ()
     echo "        TemplateApp"
     echo "        UserList"
     echo "        VFConnector"
-    echo "        SalesforceSDKTest"
-    echo "        SmartStoreTest"
-    echo "        SmartSyncTest"
```
Comment (bhariharan) [inline tools/sdk.sh]: ```ForcePluginsTest``` should still be here I think?

**Comment by bhariharan** [comment]
Otherwise, looks fine.

**Comment by wmathurin** [comment]
@bhariharan ForcePluginsTest is back

**Pull request merged**